src/Pages/Auth/Register/index.js

```diff
@@ -1,15 +1,133 @@
-import React from "react"
-import { Link } from "react-router-dom"
+import React, { useState, useEffect, useMemo } from "react"
+import { useQuery } from "react-query"
+import Input from "components/Input"
+import Button from "components/Button"
+import { useI18n } from "providers/I18n"
 
 Register.propTypes = {}
 
 function Register() {
+  const { t } = useI18n()
+  // eslint-disable-next-line
+  let userCode = useMemo(() => {
+    return window.location.href
+    // eslint-disable-next-line
+  }, [window.location.href])
```

Comment:
In order to use the location from the url we are going to use the useLocation hook from the React Router
https://reactrouter.com/web/api/Hooks/uselocation
After you get the location you will need to read the `search` portion of the url because the url will look something like this
`www.frontidapp.gr/register?pin=123456` and you will have to get the pin. 
You can also use the URLSearchParams object like this
`new URLSearchParams(location.search).get("pin")`